lisa/analysis/tasks.py

```diff
@@ -671,10 +656,10 @@ def plot_tasks_wakeups(self, target_cpus=None, time_delta=0.01, axis=None,
         if target_cpus:
             df = df[df.target_cpu.isin(target_cpus)]
 
-        df = self._df_discretize_series(df["target_cpu"], time_delta, "Wakeup count")
-
-        df.plot(ax=axis, legend=False)
+        series = series_rolling_apply(df["target_cpu"], lambda x: x.count() / (x.max() - x.min()),
```

Comment:
s/x.max() - x.min()/x.index[-1] - x.index[0]/

Since `x` could contain only one item, you'll need to handle division by 0. A dropna() before plotting should be enough to avoid having blanks segments in the graph.